### ISSUE TITLE
commit_connection_failed: uint64 addition overflow

### ISSUE THREAD

**Issue by boddumanohar** [0chain/0chain#1749]
During loadTest, there are a lot of errors on the blobbers in this format

```
"2022-10-14T10:51:02.038Z\tERROR\twritemarker/worker.go:77\tError redeeming the write marker.\t{\"allocation\": \"9faf57a3feff3f9e63b140d84f6d0ac36374696a16a9974d0977daa87f650de7\", \"wm\": \"9faf57a3feff3f9e63b140d84f6d0ac36374696a16a9974d0977daa87f650de7\", \"error\": \"transaction_verify_error: \\\"commit_connection_failed: moving tokens: can't move tokens to challenge pool: uint64 addition overflow\\\"\"}\n"
```

transaction status:
```
{
  "ID": 651266,
  "CreatedAt": "2022-10-14T10:55:08.651632Z",
  "UpdatedAt": "2022-10-14T10:55:08.651632Z",
  "DeletedAt": null,
  "hash": "57c561695ac3d7cb1c05d2cc7ed8479b21b17e910d3483b8110e1cbbb4ec85e6",
  "block_hash": "d3fc9c4bc6c5ac2168510f9063c501b466e8b91974c4c42eeaa55981eba5fe29",
  "round": 175969,
  "version": "1.0",
  "client_id": "51157bff97869618a500f87950be7c2ec4dd0f4493c8fed780c0e108bfb2467a",
  "to_client_id": "6dba10422e368813802877a85039d3985d96760ed844092319743fb3a76712d7",
  "transaction_data": "{\"name\":\"commit_connection\",\"input\":{\"allocation_root\":\"7aa771c2b44faa55c6742f68c1c7ad10b67b0365dd4b4bafa2bf71c25e686ec2\",\"prev_allocation_root\":\"1ac7aad784a470ae00b99463028be003bc427e7b07adf18709f4f002cdc80cce\",\"write_marker\":{\"allocation_root\":\"7aa771c2b44faa55c6742f68c1c7ad10b67b0365dd4b4bafa2bf71c25e686ec2\",\"prev_allocation_root\":\"1ac7aad784a470ae00b99463028be003bc427e7b07adf18709f4f002cdc80cce\",\"allocation_id\":\"9faf57a3feff3f9e63b140d84f6d0ac36374696a16a9974d0977daa87f650de7\",\"size\":67360,\"blobber_id\":\"51157bff97869618a500f87950be7c2ec4dd0f4493c8fed780c0e108bfb2467a\",\"timestamp\":1665726622,\"client_id\":\"942f45b135cd62a58697f0f5a00a1ff9617823cd6745b62e236aebbaf66e424f\",\"signature\":\"fd708c2daa90c7821824fb050b8e1f81788650b8097ef715a9494ae55d6975a4\",\"lookup_hash\":\"76a0b057941016f37adb140408be3da078c45f26c93ae39bca21f701c03c9e21\",\"name\":\"file2647641522.zcn\",\"content_hash\":\"6b43ba43c7d78ccfbbee7214e36f4c0140203ba278d486f27990f225202bf2a0\"}}}",
  "value": 0,
  "signature": "9fdd5348086b1c453a064e35112c72ba1bbac3f675dd641540f6489dd8650b85",
  "creation_date": 1665744907000000000,
  "fee": 0,
  "transaction_type": 1000,
  "transaction_output": "commit_connection_failed: moving tokens: can't move tokens to challenge pool: uint64 addition overflow",
  "output_hash": "bfef4b01ff70efa56a53048571713d572de5aefa3ffa5ee66453d6e9748436ee",
  "status": 2,
  "ReadMarkers": null,
  "WriteMarker": null
}
```

**Comment by peterlimg** [comment]
This usually happens when negative integrate value is assigned to unsigned integer variable, perhaps rewards. Perhaps is fixed in #1724 with [commit](https://github.com/0chain/0chain/pull/1724/commits/d89ea3462fa28b7d7e47f082900af4ae35ad80bb).

**Comment by dabasov** [comment]
I've seen a lot of this when we dIvided by 0

**Comment by peterlimg** [comment]
@boddumanohar @0xhrsh please check if this error still exist with staging code.

**Comment by boddumanohar** [comment]
checked in the last 2 runs, didn't see this error. Thanks for fixing this.

**Comment by peterlimg** [comment]
Should have been fixed via #1724 , reopen if we see this error again later on.
